scripts/transform.sh

```diff
@@ -59,5 +59,6 @@ cd "$individual_packages_folder"
 createSymlinks "$individual_packages_folder"
 
 if [ "$skip_build" != "true" ]; then
-  PHASE=transform yarn lerna run --stream $runtarget
+  CONCURRENCY=${CONCURRENCY:-4}
```

Comment:
Should we call this `TRANSFORM_CONCURRENCY` or some other just to make sure it doesn't conflict with some other library's env var?

Reply:
Sure, can do.